### PR TITLE
new API for export

### PR DESCRIPTION
**Pull request by xin3he**
## Type of Change

new API for export

## Expected Behavior & Potential Risk

UT pass

## How has this PR been tested?

UT

### CODE REVIEW

File: neural_compressor/adaptor/pytorch.py [hunk 1]
```diff
@@ -2918,6 +2924,11 @@ def _post_hook_for_qat(self):
             PyTorch_FXAdaptor.convert_sub_graph(self.sub_module_list, \
                                                 self.model._model, prefix='')
 
+        if self.approach != 'post_training_dynamic_quant':
+            self._get_scale_zeropoint(self.model._model, self.model.q_config)
+        self._dump_model_op_stats(self.model._model, self.model.q_config, self.approach)
+        torch_utils.util.get_embedding_contiguous(self.model._model)
```
Comment (ftian1) [inline neural_compressor/adaptor/pytorch.py:2930]: does this make sense? it looks like NLP specific func call?

**Review by PenghuiCheng — APPROVED** [review]

File: neural_compressor/adaptor/pytorch.py (hunk 1 above)
Comment (xin3he) [reply neural_compressor/adaptor/pytorch.py:2930]: All int8 model will execute these post-processes, won't have impact on cv models.

Comment (xin3he) [reply neural_compressor/adaptor/pytorch.py:2930]: this logic exists in quantize func and I just move it to hook func.

**Review by ftian1 — APPROVED** [review]

File: neural_compressor/adaptor/pytorch.py (hunk 1 above)
Comment (ftian1) [inline neural_compressor/adaptor/pytorch.py:2930]: pls add func comments to explain this

Comment (ftian1) [reply neural_compressor/adaptor/pytorch.py:2930]: @xin3he  pls submit another PR to add comments for this code

**Pull request merged**